README.md

```diff
@@ -1,13 +1,27 @@
 # GDX Looker SBC Call Centre Block
-This is for the Government of British Columbia’s SBC Call Centre LookerML project. LookerML is a language for describing dimensions, aggregates, calculations and data relationships in a SQL database
+This is the Looker project for the Government of British Columbia’s Service BC Call Centre.
 
 # Features
```

Comment:
Rephrase to describe the service delivered by the code stored in this repository

Reply:
Done